src/Tasks.java

```diff
@@ -0,0 +1,84 @@
+public class Tasks {
+
+  public void testOneXBet() {
+    oneXBet(2, 0, 2, 0);
+    oneXBet(0, 2, 0, 3);
+    oneXBet(2, 0, 2, 3);
+    oneXBet(2, 2, 2, 2);
+    oneXBet(2, 2, 4, 2);
+    oneXBet(2, 2, 3, 3);
+    oneXBet(2, 2, 1, 1);
+  }
+
+  public void testDiamondOfStar() {
+    diamondOfStar(4);
+    diamondOfStar(7);
+    diamondOfStar(8);
+  }
+
+  public void testMathCalculation() {
+    System.out.println("X в квадрате:" + mathCalculation(2,2));
+    System.out.println("X в кубе:" + mathCalculation(2,3));
+    System.out.println("X в степени N:" + mathCalculation(4,5));
+    System.out.println("X в отрицательной степени N:" + mathCalculation(2,-2));
+    System.out.println("-X в отрицательной степени N:" + mathCalculation(-2,-3));
+  }
+
+  private void oneXBet(int firstTeamResult, int secondTeamResult, int predictionOnTheFirstTeam, int predictionOnTheSecondTeam) {
+
+    int checkTotalWinOrDraw = (predictionOnTheFirstTeam == firstTeamResult && predictionOnTheSecondTeam == secondTeamResult) ? 2 :
```

Comment:
this line is too long. Should be max 100 lines: 
https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit